portal/src/main/webapp/js/src/oncokb/OncoKBConnector.js

```diff
@@ -157,6 +157,7 @@ var OncoKB = (function(_, $) {
     function Instance(id) {
         this.initialized = false;
         this.dataReady = false;
+        this.civicDataReady = false;
```

Comment:
I think in the future, each resource under annotation column will have its own dataProxy and related code into a library.